vcf-to-gds-wf.wdl

```diff
@@ -173,7 +170,7 @@ task check_gds {
 		python << CODE
 		import os
 
-		def vcf_or_gds_with_space(py_splitstring):
+		def split_n_space(py_splitstring):
```

Comment:
I think a bit of documentation (about what the function returns) would be good here.
Something along the lines of
```
Returns tuple with [file name w/ chromosome name replaced by space, The chromosome name]
for example: test_data_chrX.vcf.gz returns
["test_data_chr .vcf.gz", "X"]```